### DISCUSSION TITLE
Pre-requisities

### DISCUSSION THREAD

**Discussion by richardcase** [weaveworks/flintlock#107] [General]
## Firecracker with macvtap support

To use the macvtap functionality you will need to custom build Firecracker using the [feature/macvtap branch](https://github.com/firecracker-microvm/firecracker/tree/feature/macvtap). If you don't want to compile Firecracker then you can use the following compiled versions:
[firecracker_macvtap.zip](https://github.com/weaveworks/reignite/files/7278467/firecracker_macvtap.zip)

**Comment by Callisto13** [comment]
We have now forked firecracker to ensure the macvtap branch is kept up to date with main. (See https://github.com/weaveworks/flintlock/issues/166)

Use the [fork branch](https://github.com/weaveworks/firecracker/tree/feature/macvtap) to create a custom build or use the compiled version: 
[firecracker_macvtap_forked.zip](https://github.com/weaveworks/flintlock/files/7609728/firecracker_macvtap_forked.zip)

**Comment by Callisto13** [comment]
We are now publishing releases of our own fork: https://github.com/weaveworks/firecracker/releases
